src/interfaces/IVault.sol

```diff
@@ -5,14 +5,14 @@ interface IVault {
     // ------------------------------------------------ CUSTOM ERROR ------------------------------------------------
     error ZeroAddress();
     error UnrecognizedParam();
-    error BadHealthFactor();
+    error BadCollateralRatio();
     error PositionIsSafe();
     error ZeroCollateral();
     error TotalUserCollateralBelowFloor();
     error CollateralAlreadyExists();
     error CollateralDoesNotExist();
     error NotOwnerOrReliedUpon();
-    error HealthFactorNotImproved();
```

Comment:
same here

Reply:
can you also add a GPL-3.0 license to the project?

Reply:
> can you also add a GPL-3.0 license to the project?

done